crates/core/c8y_smartrest/src/smartrest_serializer.rs

```diff
@@ -207,13 +208,13 @@ mod tests {
     use super::*;
     use json_sm::*;
 
-    #[test]
-    fn serialize_smartrest_supported_operations() {
-        let smartrest = SmartRestSetSupportedOperations::default()
-            .to_smartrest()
-            .unwrap();
-        assert_eq!(smartrest, "114,c8y_SoftwareUpdate,c8y_LogfileRequest\n");
-    }
+    // #[test]
+    // fn serialize_smartrest_supported_operations() {
+    //     let smartrest = SmartRestSetSupportedOperations::default()
```

Comment:
I would avoid to ignore such a test even temporarily. This is just a matter of secs to replace `default` by `new` and will be great to check that parameter lifetime of the `SmartRestSetSupportedOperations` struct is not too annoying.